### PR TITLE
changed basic-types based on comments.

### PR DESCRIPTION
**Pull request by brokenprogrammer**

### CODE REVIEW

File: basics/basic-types.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ om programmet är kompilerat för ett 32-bitar eller ett 64-bitars system.
 |`real`   | >= 64-bit (generellt 64-bit, men 80-bit på Intel x86 32-bit)
 
 Prefixet `u` betecknar en *osignerad* typ. `char` översätter till
-UTF-8 tecken, `wchar` används i UTF-16 strängar och `dchar`
+ett UTF-8 tecken, `wchar` används i UTF-16 strängar och `dchar`
```
Comment (jacob-carlborg) [inline basics/basic-types.md:26]: Still mixing `char` and `string`, not sure if that matters.

Comment (brokenprogrammer) [reply basics/basic-types.md:26]: I am not entirely sure of how to translate it properly then, the english paragraph reads:

> The prefix u denotes unsigned types. char translates to UTF-8 characters, wchar is used in UTF-16 strings and dchar in UTF-32 strings.

Do you have any suggestions?

Comment (jacob-carlborg) [reply basics/basic-types.md:26]: That's the correct translation, but I would say that the English text is not consistent.

**Comment by jacob-carlborg** [comment]
There's still one "Typ egenskaper" here: https://github.com/dlang-tour/swedish/blob/f3c567f5b2795fb906d87f454d8ca3b32f74118b/basics/basic-types.md#avancerade-referenser.

**Comment by jacob-carlborg** [comment]
👌

**Pull request merged**